### PR TITLE
feat: document transformers feature

### PR DESCRIPTION
**Pull request by jiangmengyu0409**
Issues:

* https://github.com/Wiredcraft/nestjs-bunyan-logger/issues/16

Status:

- [x] add description for usage in readme
- [x] add the test code for the transformer feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,34 +50,202 @@ export class AppModule {}
...
-    @Logger() private logger: Bunyan,
-  ) {
-  }
+  constructor(@Logger() private logger: Bunyan) {}
 }
 ```
 
+## Transformers Option
+
+![image](./docs/images/transformers-workflow.png)
```
[the remark this answers is not in the document]
Comment (jiangmengyu0409) [reply README.md:79]: This is a good point, we don't have such rules for orders, and let me redraw the picture to make it more accurate.

File: README.md [hunk 2]
```diff
@@ -48,34 +50,202 @@ export class AppModule {}
...
+// The above will generate the following log entry:
+//  {
+//    originalField: 'abc',
+//    customField: '123'
+//  }
+```
+
+### Clone
+
+`clone` type allows the user to copy value from one fieldname to the specified fieldname with the following limitations:
```
[the remark this answers is not in the document]
Comment (jiangmengyu0409) [reply README.md]: Fixed in the latest commit.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiangmengyu0409) [reply README.md:79]: Fixed in the latest commit.

**Review by xavierchow — APPROVED** [review]
thx, LGTM

**Pull request merged**